src/CommunityToolkit.Maui.MediaElement/Extensions/VttParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+static partial class VttParser
+{
+	static readonly string[] separator = ["\r\n", "\n"];
+	static readonly Regex timecodePattern = new((@"(\d{2}:\d{2}:\d{2}\.\d{3}) --> (\d{2}:\d{2}:\d{2}\.\d{3})"));
+	/// <summary>
+	/// The ParseVttContent method parses the VTT content and returns a list of SubtitleCue objects.
+	/// </summary>
+	/// <param name="vttContent"></param>
+	/// <returns></returns>
+	public static List<SubtitleCue> ParseVttContent(string vttContent)
+	{
+		List<SubtitleCue> cues = [];
+		if (string.IsNullOrEmpty(vttContent))
+		{
+			return cues;
+		}
+		string[] lines = vttContent.Split(separator, StringSplitOptions.None);
+
+		SubtitleCue? currentCue = null;
+		foreach (var line in lines)
+		{
+			var match = timecodePattern.Match(line);
+			if (match.Success)
+			{
+				if (currentCue is not null)
+				{
+					cues.Add(currentCue);
+				}
+
+				currentCue = new SubtitleCue
+				{
+					StartTime = TimeSpan.Parse(match.Groups[1].Value, new CultureInfo("en-US")),
```

Comment:
can we cache this `CultureInfo`? And also, we want to force it on users?

Reply:
Updated to use `CultureInfo.InvariantCulture`